buildenv/jenkins/Jenkinsfile

```diff
@@ -1,12 +1,12 @@
 #!groovy
 pipeline {
-    agent {label "${params.MACHINE_LABEL}&&x64&&test"}
+    agent {label "${params.MACHINE_LABEL}"}
     parameters {
 		choice (choices: 'x64_linux\nx64_mac\ns390x_linux\nppc64le_linux\naarch64_linux', description: 'PLATFORM?', name: 'PLATFORM')
 		choice (choices: 'SE90\nSE80', description: 'What is JAVA_VERSION?', name: 'JAVA_VERSION')
 		choice (choices: 'openjdk9-openj9\nopenjdk8-openj9\nopenjdk9\nopenjdk8', description: 'What is JVM Version?', name: 'JVM_VERSION')
-		choice (choices: 'openjdk\nsanity\nextended\nruntest', description: 'What is test level?', name: 'TARGET')
-		choice (choices: 'linux\nmac', description: 'What is machine label?', name: 'MACHINE_LABEL')
+		choice (choices: 'openjdk\nsanity\nextended\ntest_systemtest\nruntest', description: 'What is test level?', name: 'TARGET')
+		choice (choices: 'linux&&x64&&test\nopentest&&linux\nmac', description: 'What is machine label?', name: 'MACHINE_LABEL')
```

Comment:
@sophia-guo may have already done this tidy up for the hard-coded machine label, please be aware as you may have to update to grab that recent change

Reply:
Thanks for pointing this out. I have merged in Sophia's change and refreshed the PR.